docs/core/idioms/flow-to-flow.md

```diff
@@ -32,34 +30,13 @@ from prefect.tasks.prefect import create_flow_run
 def extract_some_data():
     return {"param-key": "some-random-piece-of-data"}
 
-
-# assumes you have registered a flow named "example" in a project named "examples"
-flow_run = create_flow_run(flow_name="example", project_name="examples")
-
 with Flow("parent-flow") as flow:
-    flow_run(parameters=extract_some_data)
+    params = extract_some_data()
+    # assumes you have registered a flow named "example" in a project named "examples"
+    flow_run = create_flow_run(flow_name="example", 
+                               project_name="examples", 
+                               parameters=params)
```

Comment:
I feel like you'd more often have `extract_some_data` return `"some-random-piece-of-data"` then provide the key when you create the flow run e.g. `parameters={"param-key": extract_some_data()}`

Reply:
Ah ok can make the fix

Reply:
This has been added